createAccountJava/src/main/java/com/brandon/createaccountjava/viewmodel/CreateAccountUiStates.java

```diff
@@ -0,0 +1,8 @@
+package com.brandon.createaccountjava.viewmodel;
+
+public enum CreateAccountUiStates {
+    IDLE,
+    LOADING,
+    DUPLICATE,
+    SUCCESS
```

Comment:
How is this a UI state? What state of the UI does this serve? I think you may be more served with a `state` stream and an `event` stream

Reply:
But we can talk about this

Reply:
Reworked this, let me know if that is acceptable